roles/etcd/meta/main.yml

```diff
@@ -18,3 +18,4 @@ galaxy_info:
 dependencies:
 - { role: os_firewall }
 - { role: etcd_common }
+- { role: etcd_server_certificates }
```

Comment:
In that case, should we make the CA role more generic, say `openssl_ca` so that it can be used by an *_certificates role that uses an openssl certificate authority?